rpmbuild/copr_rpmbuild/providers/scm.py

```diff
@@ -118,7 +118,7 @@ def get_make_srpm_command(self):
 
         makefile_path = os.path.join(mock_repodir, '.copr', 'Makefile')
         make_srpm_cmd_part = MAKE_SRPM_TEPMLATE.format(mock_cwd, makefile_path,
-                mock_resultdir, mock_spec_path, mock_repodir)
+                mock_resultdir, mock_spec_path, mock_workdir)
```

Comment:
This is weird though; the mock_repodir should be the directory containing `.git` subdir, right?

Reply:
that's just remnant to make it work. Actually I want mock_repodir and do:
```
[safe]
  directory = {4}
  directory = {4}/*
```